.gitignore

```diff
@@ -1,2 +1,3 @@
 /.eslintcache
 /node_modules/
+docs/.vuepress/dist
```

Comment:
```suggestion
docs/.vuepress/dist

```

Reply:
?

Reply:
Plain text files must end in an empty return to prevent EOL issues in git.

Reply:
Perfect